lib/ansiblelint/test/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""Test utils for ansible-lint."""
```

Comment:
Naming the package "test" suggests that it contains tests. I'd like it to be "testing" so that it feels like test helpers/framework instead.